cert-data.yml

```diff
@@ -12,6 +187,19 @@ edX/DemoX_v3/Demo_Course_v3:
     Pass: 'This person passed.'
     Distinction: 'This person is a complete overachiever!'
 
+edX/DemoX_v4/Demo_Course_v4:
+  <<: *cert_defaults
+  instructors:
+    - instructor:
+        <<: *instructor_1
+        name: "<b>M. Guillaume Hébert</b>"
```

Comment:
It's a play on the "Head Concierge of the Grand Budapest Hotel" :) 

Just wanted some silly name that would also have UTF8 char in it, and for which that I could safely make a bogus signature...